common/thorhelper/thorstats.cpp

```diff
@@ -20,34 +20,63 @@
 #include "thorstats.hpp"
 #include "jdebug.hpp"
 
+// Available sets of statistics for a nested section
+const StatisticsMapping defaultNestedSectionStatistics({StCycleLocalExecuteCycles, StTimeLocalExecute, StNumStarts, StNumStops});
+const StatisticsMapping genericCacheNestedSectionStatistics({StCycleLocalExecuteCycles, StTimeLocalExecute, StNumStarts, StNumStops, StNumCacheAdds, StNumCacheHits, StNumPeakCacheObjects});
```

Comment:
genericCacheNestedSectionStatistics should build on defaultNestedSectionStatistics so that any changes in defaultNestedSectionStatistics are reflected in genericCacheNestedSectionStatistics:

const StatisticsMapping genericCacheNestedSectionStatistics(defaultNestedSectionStatistics, {StNumStops, StNumCacheAdds, StNumCacheHits, StNumPeakCacheObjects});